api/routes/userRouter.js

```diff
@@ -10,8 +10,16 @@ router.get('/:id', (req, res, next) => {
 router.post('/', async (req, res, next) => {
     try {
         const {data} = req.body;
-        user = await userModel.createUser(data);
-        res.status(200).json(user);
+
+        const requiredParams = ['username', 'password', 'first_name', 'last_name', 'phone'];
+        const missingRequiredParams = requiredParams.filter(param => !(param in req.body));
+
+        if (missingRequiredParams.length > 0) {
```

Comment:
I think were going to need to change this logic bc sign in is also a post request and only takes two params. I'll just approve it here though, merge yours to main pull that and change it as I work on mine.